setup/base/base.go

```diff
@@ -478,13 +486,30 @@ func (b *BaseDendrite) SetupAndServeHTTP(
 
 	b.configureHTTPErrors()
 
+	//Redirect for Landing Page
+	externalRouter.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		http.Redirect(w, r, httputil.PublicStaticPath, http.StatusSeeOther)
+	})
+
 	internalRouter.PathPrefix(httputil.InternalPathPrefix).Handler(b.InternalAPIMux)
 	if b.Cfg.Global.Metrics.Enabled {
 		internalRouter.Handle("/metrics", httputil.WrapHandlerInBasicAuth(promhttp.Handler(), b.Cfg.Global.Metrics.BasicAuth))
 	}
 
 	b.ConfigureAdminEndpoints()
 
+	tmpl := template.Must(template.ParseFS(staticContent, "static/*.gotmpl"))
+
+	b.PublicStaticMux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		if err := tmpl.ExecuteTemplate(w, "index.gotmpl", map[string]string{
+			"Version": internal.VersionString(),
```

Comment:
It seems inefficient to template this every time the client asks. Just do it once at startup and return those bytes?